Sources/Testing/Events/Recorder/Event.HumanReadableOutputRecorder.swift

```diff
@@ -281,6 +307,18 @@ extension Event.HumanReadableOutputRecorder {
           testData.issueCount[issue.severity] = issueCount + 1
         }
         context.testData[id] = testData
+      
+      case .testCaseStarted:
+        guard verbosity >= 2 else { break }
+        let id: [String] = if let test {
```

Comment:
We don't check verbosity in this first `switch` intentionally so that the data we collect is always comprehensive. The second `switch` below checks verbosity before producing any messages.

`test` will never be `nil` for this event, and `.testStarted` will always be recorded before `.testCaseStarted`, so you can just write:

```swift
case .testCaseStarted:
  let test = test!
  context.testData[test.id.keyPathRepresentation]?.testCasesCount += 1
```